Add badges and a contributing section to readme (#131)
FIUS/ICGE2: README.md

The second version of the introduction course game engine for the Java introduction course which is organized yearly by [FIUS](https://fius.informatik.uni-stuttgart.de/) at the [University Stuttgart](https://www.uni-stuttgart.de/).

The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).

## Code format
The code in this project must conform to a specific format which is enforced for all pull requests.
The formatter can be run locally with the following console command:
```
mvn formatter:format
```

## Found a bug? / Need a feature?
Please report them by [creating an issue](https://github.com/FIUS/ICGE2/issues/new) on the [github issue page](https://github.com/FIUS/ICGE2/issues).
Please include a full stacktrace and information to reproduce the error in case of a bug report.

## Contributing
This project is managed by students part of [FIUS](https://fius.informatik.uni-stuttgart.de/).
Nevertheless help is always highly appreciated.
New features and bug fixes can be easily submitted as a pull request.

> But to avoid total chaos there are a few rules a pull request must adhere to.

Reviewer: Avoid referring to total chaos. Just say that there are some rules.

A pull request …
 -  … should state clearly what it does or fixes.
 -  … must pass all CI checks (e.g. unit tests, licence header tests and code format tests).
 -  … must be reviewed by at least one of the maintainers.
 -  … must be merged by one of the maintainers.
 -  … may be reviewed by others.
 -  … may be opened for work which is in progress but this must be opened as a draft.
 -  … should have a reference to all relevant issues.<br>

>     If it fixes them please add `Fixes #xxx` or `Closes #xxx` so that the issue is closed automatically.

Reviewer: ```suggestion
    If it fixes them please add `Fixes #xxx` or `Closes #xxx` so that the issue is closed automatically when the pull request is merged.
```

 -  … should state the motivation for this pull request. (Can be a reference to an issue.)
 -  … must adhere to  the following styling guidelines:

>      -  … must use `camel case` for variables and class attributes

Reviewer: Format cases to be example for themself

     -  … must use `pascal case` for classes
     -  … must use `snake case` for packages
     -  … must use `upper snake case` for constants

>  -  … must contain Javadoc on all non private methods and fields.

Reviewer: ```suggestion
 -  … must contain Javadoc for all non private methods and fields.
```

>  -  … should contain Javadoc on all private methods.

Reviewer: ```suggestion
 -  … should contain Javadoc for all private methods.
```

![GitHub release (latest by date including pre-releases)](https://img.shields.io/github/v/release/FIUS/ICGE2?include_prereleases&style=plastic)
![GitHub](https://img.shields.io/github/license/FIUS/ICGE2?style=plastic)

The second Version of the Introduction Course Game Engine for the Java Introduction Course which is organized yearly by the [FIUS](https://fius.informatik.uni-stuttgart.de/) at the [University Stuttgart](https://www.uni-stuttgart.de/).

## Code format
The code in this project needs to conform to a specific format.
This format can be archieved by running the formater with the console command:
```
mvn formatter:format
```

## Contributing
Since this project is done by students in their spare time, help is always highly appreciated. This as expected is done via a pull request to this repository. But to avoid total chaos there are a few rules a pull request must adhere to. A pull request …

>  - … should state clearly what it does or fixes.

Reviewer: Also:
```
- … should have a reference to all relevant issues.
- … should state the motivation for this pull request. (Can be a reference to an issue.)
```

>  - … must pass the CI checks which runs the unit tests and checks the code formating.

Reviewer: ```suggestion
 - … must pass all CI checks (e.g. unit tests, licence header tests and code format tests).
```

>  - … must be reviewed and merged by one of the core developers.

Reviewer: ```suggestion
 - … must be reviewed by at least one of the current maintainers.
 - … can only be merged by one of the current maintainers.
```

>  - … may be reviewd by others or discussed in person.

Reviewer: ```suggestion
 - … may be reviewed by others.
 - … should be opened as draft pull request if it is work in progress.
```

 - … must use camel case for variables and class attributes
 - … must use pascal case for classes
 - … must use snake case for packages

>  - … must use upper snake case for constants

Reviewer: ```suggestion
 - … must adhere to  the following styleguide guidelines:
     - must use `camelCase` for variables and class attributes
     - must use `PascalCase` for classes
     - must use `snake_case` for packages
     - must use `UPPER_SNAKE_CASE` for constants
 - … must document all non private methods and fields.
 - … private methods (except one-liners, e.g. getter and setter) should be documented with javadoc.
 - … should not introduce new eclipse warnings. Warnings should be specifically suppressed if they should not be fixed.
 - … should contain unit tests for new or changed behaviour.
```

> _Everything described here follows the word definitions of the [RFC 2119](https://tools.ietf.org/html/rfc2119)_

Reviewer: ```suggestion
The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119)
```
Maybe move this disclaimer to before the first use of a keyword.
